File: 003FizzBuzz/Ruby/fizzbuzz.rb

FIRST_VALUE = 1
LAST_VALUE = 100

def fizzbuzz
    for num in FIRST_VALUE..LAST_VALUE
        if num%15 == 0
            print "FizzBuzz\n"
        elsif num%3 == 0
            print "Fizz\n"
        elsif num%5 == 0
            print "Buzz\n"
        else
            print num.to_s + "\n"
        end
    end
end

fizzbuzz

File: 003FizzBuzz/Perl/fizzbuzz.pl

use strict;

my $FirstValue = 1;
my $LastValue = 100;

&fizzbuzz();

sub fizzbuzz {
    foreach my $num ($FirstValue .. $LastValue){
        if($num%15==0){
            print "FizzBuzz\n";
        }
        elsif($num%3==0){
            print "Fizz\n";
        }
        elsif($num%5==0){
            print "Buzz\n";
        }
        else{
            print $num."\n";
        }
    }
}
